lib/data_update_scripts/20201209222407_remove_collective_noun_from_config.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveCollectiveNounFromConfig
+    def run
+      SiteConfig.find_by(var: "collective_noun")&.or(SiteConfig.find_by(var: "collective_noun_disabled"))&.destroy
```

Comment:
We should use `.where()` here so that you can use `.or()` correctly and avoid using the `&` operator, if `collective_noun` doesn't exist, the second `OR` is not executed:

```shell
[1] pry(main)> SiteConfig.find_by(var: "collective_noun")&.or(SiteConfig.find_by(var: "collective_noun_disabled"))&.destroy
  User Exists? (52.5ms)  SELECT 1 AS one FROM "users" LIMIT $1  [["LIMIT", 1]]
  SiteConfig Load (7.4ms)  SELECT "site_configs".* FROM "site_configs" WHERE "site_configs"."var" = $1 LIMIT $2  [["var", "collective_noun"], ["LIMIT", 1]]
```

```suggestion
      SiteConfig.where(var: "collective_noun").or(SiteConfig.where(var: "collective_noun_disabled")).destroy_all
```

as you can see, this `OR` searches for both.

You can also use a simpler syntax:

```suggestion
      SiteConfig.where(var: %w[collective_noun collective_noun_disabled]).destroy_all
```

Reply:
Thank you so much for bringing this to my attention and for breaking it all down for me! I really like the simplified syntax and went with that suggestion. 😊